README.md

```diff
@@ -1,6 +1,4 @@
-[![Build Status](https://travis-ci.org/mellanox/accelerated-bridge-cni.svg?branch=master)](https://travis-ci.org/mellanox/accelerated-bridge-cni) [![Go Report Card](https://goreportcard.com/badge/github.com/mellanox/accelerated-bridge-cni)](https://goreportcard.com/report/github.com/mellanox/accelerated-bridge-cni)
```

Comment:
why you remove it we just need to create and updated it with new links

Reply:
I assumed that we have plan to switch from Travis to GitHub actions. Let's handle CI configuration as a separate PR.
I dropped this change from the PR

Reply:
For now I simply replaced org in badges path to new one. Goreport card should work, Travis will probably need a fix/configuration. I propose to do a CI configuration in another PR

Reply:
travis should work for it IMO, just need to update the env var containing the docker login credentials

Reply:
fixing CI can be in a different PR IMO